park-dashboard/src/Pages/Generic Charts/GenericCharts.js

```diff
@@ -35,6 +35,14 @@ const GenericChart = () => {
           </Col>
         </Row>
       </Container>
+      <Container style={{ marginBottom: "5rem", marginTop: "12rem"}}>
+        <Col>
+          <h2 style={{ fontSize: "18px" }} className="text-center">
+            3D Pie Chart: Alerts Vs Park Counts
+          </h2>
+          <AlertsVsParks />
```

Comment:
I am seeing a crash when I tap on `Park Closure`. Can you fix it?

Steps to reproduce:

- Go to Generic charts page
- Scroll down to `AlertsVsParks` component
- Tap on `Park Closure` 

<img width="1504" alt="image" src="https://github.com/rutujapadgilwar/Park-Dashboard/assets/114516950/a1b1235f-aaee-46fc-bb31-067942640490">